CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add ping/pong to screen sharing start code path to ensure socket is viable
 - Enable integration tests for safari 12
 - Write timestamp for latest canary completion time
+- Add basic component library setup
```

Comment:
Move this to `## [Unreleased]` section (line 14)

Reply:
Will do.